### ISSUE TITLE
Doesn't clean up `fs` properly

### ISSUE THREAD

**Issue by sindresorhus** [tapjs/stack-utils#14]
In AVA:

``` js
import fs from 'fs';
import test from 'ava';

test.cb(() => {
    fs.exists('foo', () => {
        throw new Error('123');
    });
});
```

Results in:

```
❯ ava


  2 exceptions

  1. Uncaught Exception
  Error: 123
    test.js:6:9
    FSReqWrap.cb [as oncomplete] (fs.js:212:19)


  2. Test results were not received from test.js
```

The `FSReqWrap.cb [as oncomplete] (fs.js:212:19)` part should have been cleaned up.

**Comment by jamestalmage** [comment]
The [list of ignored modules](https://github.com/tapjs/stack-utils/blob/master/index.js#L14-L26) was just copied from what `node-tap` had originally. I think I agree that `fs` should be added to the list, but while we are at it, there are a LOT of internal modules missing from that list: https://github.com/nodejs/node/tree/master/lib

What other ones should be included? All of them?

**Comment by sindresorhus** [comment]
All of them.

**Comment by sindresorhus** [comment]
Node.js also have many internal modules now, in an `internal` namespace:

```
_combinedTickCallback (internal/process/next_tick.js:67:7)
process._tickCallback (internal/process/next_tick.js:98:9)
```

Which we should also handle.

The above is from: https://github.com/avajs/ava/issues/990

**Comment by isaacs** [comment]
I pulled this list out of mocha's reporters, if I remember correctly.

The way to do this consistently would be to remove any file in `Object.keys(process.binding('natives'))`.

**Comment by isaacs** [comment]
Fixed on 1.0.0
